nest-demo/docker/tempo.yml

```
@@ -0,0 +1,60 @@
+        send_exemplars: true
+
+storage:
+  trace:
+    backend: local # backend configuration to use
+    wal:
+      path: /tmp/tempo/wal # where to store the the wal locally
+    local:
+      path: /tmp/tempo/blocks
+
+overrides:
+  metrics_generator_processors: [service-graphs, span-metrics, 'local-blocks'] # enables metrics generator  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)